examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -70,7 +70,7 @@
     "\n",
```

Comment:
<p>can we make the hidden dimension a hyperparameter variable and use it? e.g. hiddem_dim or d_dim ? so it is easy to stop which values have to be the same?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>